explorer/rest/rest/db/NemDatabase.py

```diff
@@ -39,15 +39,16 @@ def get_block(self, height):
 
 			return self._create_block_view(result) if result else None
 
-	def get_blocks(self, limit, offset, min_height):
+	def get_blocks(self, limit, offset, min_height, sort):
 		"""Gets blocks pagination in database."""
 
 		with self.connection() as connection:
 			cursor = connection.cursor()
-			cursor.execute('''
+			cursor.execute(f'''
 				SELECT *
 				FROM blocks
 				WHERE height >= %s
+				ORDER BY height {sort}
```

Comment:
is there a situation where blocks are added to the db out of order? if not, it should be faster to sort by id instead of height

Reply:
nope, the block is inserted in order actually.

I just found out, my block table didn't have id 😬 
I will update my table.